manim/renderer/shaders/include/NOTE.md

```diff
@@ -0,0 +1,7 @@
+There seems to be no analog to #include in C++ for OpenGL shaders.  While there are other options for sharing code between shaders, a lot of them aren't great, especially if the goal is to have all the logic for which specific bits of code to share handled in the shader file itself.  So the way manim currently works is to replace any line which looks like 
+
+#INSERT <file_name>
```

Comment:
isn't it `include`?

Reply:
That construct isn't standardized; we have to search for and resolve the includes by hand.